index.html

```diff
@@ -88,9 +106,11 @@ <h3 class="work-header">Multi-Post Stories</h3>
               <li class="tag ruby-tag"><a href="#">Ruby</a></li>
             </ul>
             <button type="button" class="work-card-button">See Project</button>
```

Comment:
I apologize if no reviewer previously mentioned this, but some of your buttons have a `border-radius` that is not present in the template design.

I'm marking just this one button for brevity, but you should fix all of your buttons.